packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GetOauthTokenTask.java

```diff
@@ -0,0 +1,87 @@
+package io.flutter.plugins.googlesignin;
+
+import android.accounts.Account;
+import android.content.Context;
+import android.os.AsyncTask;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+import com.google.android.gms.auth.GoogleAuthUtil;
+import java.util.List;
+
+/** Android inbuilt solution for calling async tasks */
```

Comment:
Please describe what instances of the class represent, rather than the implementation tech.
E.g. "Task for retrieving an OAuth token in the background."?